guides/common/modules/proc_enabling-pull-mode-on-proxy.adoc

```diff
@@ -0,0 +1,25 @@
+[id="enabling-pull-mode-on-proxy_{context}"]
```

Comment:
Maybe also use attribute instead of "proxy" if available?

Reply:
Should this be done in bulk in another PR perhaps?

Reply:
@Lennonka Yes, of course.